card-1/css/_container.scss

```
@@ -0,0 +1,106 @@
+.top-img{
```

```suggestion
.top-img {
```
Mozda bi instalacija prettier-a resila probleme sa ovakvim greskama. Ima ih jos u fajlu....